web/src/OverviewResourceSidebar.stories.tsx

```diff
@@ -53,3 +54,21 @@ export function TwoResourcesTwoTests() {
   let view = { resources: all, tiltfileKey: "test" }
   return <OverviewResourceSidebar name={""} view={view} />
 }
+
+export function TestsWithErrors() {
+  let all: Resource[] = [tiltfileResource()]
+  for (let i = 0; i < 8; i++) {
+    let test = oneResourceTest()
+    test.name = "test_" + i
+    if (i % 2 === 0) {
+      if (!test.buildHistory) {
+        throw "unexpectedly empty buildHistory on test resource"
+      }
+      test.buildHistory[0].error = "egads!"
```

Comment:
TIL you can cut the if statement and use (the non-null assertion operator)[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator], i.e., `test.buildHistory![0].error = "egads!"`

Reply:
Nice tip, thanks for helping me grok this!